modules/cnv_configuring_masquerade.adoc

```diff
@@ -0,0 +1,35 @@
+[[configuring-masquerade]]
+=== Configuring masquerade mode
+
+You can use masquerade mode to hide a virtual machine's outgoing traffic behind
+the pod IP address. Masquerade mode uses Network Address Translation (NAT) to
+connect virtual machines to the pod network backend through a Linux bridge.
+
+Enable masquerade mode and allow traffic to enter the virtual machine by
+editing your virtual machine configuration file.
+
+.Prerequisites
+
+* The virtual machine must be configured to use DHCP to acquire IPv4 addresses.
+
+.Procedure
+
+. Edit the `interfaces` spec of your virtual machine configuration file:
```

Comment:
A user should know how to find this, right?

Reply:
The file? Yes.